keras_eval/visualizer.py

```diff
@@ -181,42 +181,48 @@ def plot_threshold(threshold, correct, errors, title='Threshold Tuning'):
     iplot(fig, filename='Threshold Tuning')
 
 
-def plot_images(image_paths, n_imgs, title='', image_res=(20, 20), save_name=None):
+def plot_images(image_paths, n_images, title='', n_cols=5, image_res=(20, 20), save_name=None):
     n_row = 0
     n_col = 0
 
-    if n_imgs <= 5:
-        f, axes = plt.subplots(nrows=1, ncols=n_imgs, figsize=image_res)
-        plt.title(title)
-        for i, image_path in enumerate(image_paths):
-
-            if i == n_imgs:
-                break
+    if len(image_paths) < n_images:
```

Comment:
Maybe we could display a message like "The total number of requested images are not present are not present in the given path" or something along these lines, so that there is no discrepancy between `n_images` and number of plots displayed.

Reply:
Number of plots displayed will be min(len(image_paths, n_images))

Reply:
I know, but that is not obvious to the end user.

Reply:
Hope it is clearer now.